coi/src/stats.rs

```diff
@@ -123,6 +124,31 @@ pub fn compute_coi_decay_factor(horizon: Duration, now: SystemTime, last_view: S
     ((horizon - days) / (horizon - 1. - f32::EPSILON)).max(0.)
 }
 
+/// Computes a weight distributions across cois based on their relevance.
+pub fn compute_coi_weights(cois: &[PositiveCoi], horizon: Duration) -> Vec<f32> {
```

Comment:
moved here as it's kinda the same kind of function/deps situations as `compute_coi_relevance` (which is just above this function)